blueprint/office-365.md

```diff
@@ -17,6 +17,9 @@ Power Platform | The Power Platform includes PowerApps, Power BI, and Microsoft
 Security and Compliance | Office 365 provides Security and Compliance tools which can be utilised to implement an organisation's Information Management Policy and to assist with information governance.
 Exchange Online Protection | Exchange Online protection is a cloud hosted email security service (Mail Gateway) that acts to filter spam and scan for viruses on email entering and leaving Exchange Online.
 Office 365 Advanced Threat Protection (ATP) | Office 365 ATP is a cloud-based mail threat protection service. The service provides protection against unknown malware and viruses through the use of robust zero-day protection and inclusion of features to safeguard an organisation from harmful links in real time.
+Microsoft Whiteboard | Microsoft Whiteboard is a digital whiteboard that facilitates collarboration between users. It has integrations with the Office 365 suite and physical endpoints such as the Microsoft Surface hub.
```

Comment:
Suggest running this file through a spell checker. There's a couple, eg collarboration